packages/alert/__tests__/Alert.test.tsx

```diff
@@ -54,20 +67,31 @@ describe('with props.variant', () => {
             {variant}
           </Alert>,
         )
-        expect(document.querySelectorAll('span')).toHaveLength(1)
         expect(document.querySelectorAll(`[fill="${colorMapping[variant]}"`)).toHaveLength(1)
       }
     },
   )
 })
 
 describe('with props.type', () => {
-  it.each<AlertProps['type']>(['toast', 'banner'])('renders alert with type', (type) => {
-    if (type) {
-      render(<Alert description="alert description" type={type} />)
-      expect(document.querySelectorAll('span')).toHaveLength(1)
-    }
-  })
+  it.each<AlertProps['type']>(['toast', 'banner', 'relative'])(
+    'renders alert with type',
+    (type) => {
+      if (type) {
+        render(
+          <Alert
+            description="alert description"
+            type={type}
+            showCloseButton={false}
+            isOpen={true}
+            offsetIndex={0}
+            delay={3000}
```

Comment:
It doesn't look like we are testing delay here. We better remove it to avoid confusion.